Products/ZenStatus/nmap/util.py

```diff
@@ -0,0 +1,146 @@
+##############################################################################
```

Comment:
flake8 feedback:


 Line Col Level    ID     Message (Checker)                                                                                                                                                                                                                                   
   13   1 warning  F401   'os' imported but unused... (python-flake8)                                                                                                                                                                                                         
   20   1 warning  F401   'binPath' imported but unused... (python-flake8)                                                                                                                                                                                                    
   50   1 warning  C901   'executeNmapCmd' is too complex (13)... (python-flake8)                                                                                                                                                                                             
  119  31 warning  E231   missing whitespace after ','... (python-flake8)